metadata/rules/gorule-0000031.md

```diff
@@ -11,18 +11,9 @@ implementations:
     source: https://github.com/biolink/ontobio/blob/master/ontobio/io/assocparser.py#L390
 ---
 ## In GAFs, relation is overloaded into qualifier.
```

Comment:
My only comment is we should probably take out the `##` in this line and in the following lines. These produce headings in markdown and when rendered this file looks pretty strange.